### PR TITLE
Feature/desktop-View

### PR DESCRIPTION
**Pull request by Yordinia**
# Feature / Desktop-View

- Implemented desktop view Figma design into your mobile view portfolio project
- Checked linter errors
- Used template three

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -296,116 +348,278 @@ a:active {
...
+
+  .detail-nav ul {
+    margin: 0 auto;
+    position: fixed;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 8px;
+    justify-content: center;
+  }
```
Comment (Favourezeugwa) [inline style.css]: I think your content of your portfolio is scrolling into your `navbar`. 
This is a screenshot of your webpage.
Screenshot 2022-11-12 at 13 53 04
Can we fix this? 

Here is my suggestion: 👇🏽 
```
  .nav {
    display: block;
    padding: 0px 50% 0px 50%;
    width: 100%;
    position: fixed;
    background-color: white;
    align-items: center;
  }

```
Kindly add this code snippet to your .nav selector in your media query then delete line 479 (`margin: 0 auto`) and 480 (`position: fixed`) in the in-line comment highlighted above. 🤗  

I think your `.detail-nav ul` selector should look like this: 👇🏽 

```
 .detail-nav ul {
    display: flex;
    flex-direction: row;
    align-items: center;
    gap: 8px;
    justify-content: center;
  }
```

**Review by Favourezeugwa — CHANGES_REQUESTED** [review]
## STATUS: REQUIRES CHANGES 🚦 ⚠️ ♻️ 

Hi @Yordinia , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-brianna


## Highlights 💯 
- No Linter errors ✔️
- Descriptive Pull request summary✔️
- Good commit messages ✔️
- Professional Readme file ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by pinkmoon25** [comment]
Hi @Yordinia , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### **You have submitted a project, that  does not pass linter checks, your stylelint is failing please fix it and submit again for a review.**

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 2]
```diff
@@ -1,54 +1,77 @@
 * {
   margin: 0;
   padding: 0;
+
+  /* outline: 2px solid #f80606;  */
```
Comment (christianbiring1) [inline style.css:5]: [OPTIONAL]
- [x] Kindly consider removing the commented block of code in your code base as it's not the best practice for clean code.

File: style.css [hunk 3]
```diff
@@ -296,116 +344,274 @@ a:active {
...
+  margin: auto;
+  gap: 58px;
+  width: 100%;
 }
 
-form {
-  background-color: rgb(255, 255, 255) solid;
-  max-width: 327px;
-  padding: 0.1rem 0 0 0;
-  margin: 0 auto;
```
Comment (christianbiring1) [inline style.css]: [OPTIONAL]
- [ ] Kindly make sure you stick to the design as much as possible.  (font, spacing(`margin, padding`), color ...
- [ ] Kindly you may consider centering the last section of the page. (the form and the side literature.) `hint: You may consider using the flexbox or the grid container to do so.`
- [ ] Also make sure there's no scroll bar at the bottom of the page. the page content has to fit the hundred percent width of the screen size
Your design :point_down: 
image

image


The Figma template :point_down: 
image

**Review by christianbiring1 — APPROVED** [review]
Hi @Yordinia , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
Iapproveokay

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] Good commit history.
## Optional suggestions
- [ ] `I HIGHLY RECOMMEND YOU IMPLEMENT THEM FOR THEY WILL TAKE YOUR PROJECT TO THE NEXT LEVEL` :rocket: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**